
----------------------------------------
{'2025-06-13 11:01:29'} - Fortigate IP: 100.64.14.14 - Port: 15443
Fortigate IP: 100.64.14.14
No API key provided, cannot perform API tests.
Fortigate IP: 100.64.14.14
No username provided, cannot log in.

----------------------------------------
{'2025-06-13 11:06:22'} - Fortigate IP: 100.64.14.14 - Port: 15443
Response data:
Test 1 URL: https://100.64.14.14:15443/api/v2/monitor/system/status

Test 1 Error: HTTPSConnectionPool(host='100.64.14.14', port=15443): Max retries exceeded with url: /api/v2/monitor/system/status (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x00000202F9B01290>, 'Connection to 100.64.14.14 timed out. (connect timeout=None)'))Response Data: API call failed with the keyFortigate IP: 100.64.14.14
No username provided, cannot log in.

----------------------------------------
2025-06-13 11:07:37 - Fortigate IP: 100.64.14.14 - Port: 15443
Response data:
Test 1 URL: https://100.64.14.14:15443/api/v2/monitor/system/status

Test 1 Error: HTTPSConnectionPool(host='100.64.14.14', port=15443): Max retries exceeded with url: /api/v2/monitor/system/status (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000002098C17F810>, 'Connection to 100.64.14.14 timed out. (connect timeout=None)'))Response Data: API call failed with the keyLogin failed, cannot perform API tests.

----------------------------------------
2025-06-13 11:09:16 - Fortigate IP: 100.64.14.14 - Port: 15443
Response data:
Test 1 URL: https://100.64.14.14:15443/api/v2/monitor/system/status

Test 1 Error: HTTPSConnectionPool(host='100.64.14.14', port=15443): Max retries exceeded with url: /api/v2/monitor/system/status (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x00000204E33BF990>, 'Connection to 100.64.14.14 timed out. (connect timeout=None)'))Response Data: API call failed with the keyLogin failed, cannot perform API tests.

----------------------------------------
2025-06-13 11:18:26 - Fortigate IP: 100.64.14.14 - Port: 15443
Test 1 URL: https://100.64.14.14:15443/api/v2/monitor/system/status
Test 1 Error: HTTPSConnectionPool(host='100.64.14.14', port=15443): Max retries exceeded with url: /api/v2/monitor/system/status (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x00000145A768F590>, 'Connection to 100.64.14.14 timed out. (connect timeout=None)'))Response Data: API call failed with the keyLogin failed, cannot perform API tests.

----------------------------------------
2025-06-13 11:27:00 - Fortigate IP: 100.64.14.14 - Port: 15443
Test 1 URL: https://100.64.14.14:15443/api/v2/monitor/system/status
Test 1 Error: HTTPSConnectionPool(host='100.64.14.14', port=15443): Max retries exceeded with url: /api/v2/monitor/system/status (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x00000207E6673D50>, 'Connection to 100.64.14.14 timed out. (connect timeout=None)'))
Response Data: API call failed with the key
Login failed, cannot perform API tests.

----------------------------------------
2025-06-13 11:36:32 - Fortigate IP: 100.64.14.14 - Port: 15443
Test 1 URL: https://100.64.14.14:15443/api/v2/monitor/system/status
Test 1 Error: HTTPSConnectionPool(host='100.64.14.14', port=15443): Max retries exceeded with url: /api/v2/monitor/system/status (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000001E57CF5F490>, 'Connection to 100.64.14.14 timed out. (connect timeout=None)'))
Response Data: API call failed with the key
Login failed, cannot perform API tests.